infra/workbooks/workbook.json

```diff
@@ -6,11 +6,1710 @@
       "content": {
         "json": "# Chat With Your Data Monitoring"
       },
-      "name": "heading"
+      "name": "Heading"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "crossComponentResources": [
+          "{subscription-id}"
+        ],
+        "parameters": [
+          {
+            "id": "b958a893-1fec-49c0-9487-5404949fa49d",
+            "version": "KqlParameterItem/1.0",
+            "name": "appserviceplan",
+            "type": 5,
+            "isRequired": true,
+            "isGlobal": true,
+            "query": "resources\r\n| where type =~ 'Microsoft.Web/serverfarms'\r\n| summarize by id",
+            "crossComponentResources": [
+              "{subscription-id}"
+            ],
+            "isHiddenWhenLocked": true,
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources",
+            "value": "{subscription-id}/resourceGroups/{resource-group}/providers/Microsoft.Web/serverFarms/{app-service-plan}"
```

Comment:
based on my comment above, could this then become {hostingplan.outputs.id}

Reply:
Ah yes it could be. I started off that way, then undid it when I reached the `-docker` deployments, then forgot to put it back when we decided to keep the `-docker` 🤦